app/src/test/java/com/stevesoltys/seedvault/plugins/webdav/WebDavTestConfig.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2024 The Calyx Institute
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.stevesoltys.seedvault.plugins.webdav
+
+import org.junit.Assume.assumeFalse
+import org.junit.jupiter.api.Assertions.fail
+
+object WebDavTestConfig {
+
+    fun getConfig(): WebDavConfig {
+        assumeFalse(System.getenv("NEXTCLOUD_URL").isNullOrEmpty())
+        return WebDavConfig(
```

Comment:
Another option could be something like WireMock if we didn't want to actually depend on an external service for tests.

But this is probably easier and likely provides better coverage anyways..